acceptance/framework/consul/helm_cluster.go

```diff
@@ -222,6 +223,66 @@ func (h *HelmCluster) Destroy(t *testing.T) {
 			}
 		}
 
+		// Delete any deployments that have h.releaseName in their name.
+		deployments, err := h.kubernetesClient.AppsV1().Deployments(h.helmOptions.KubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{LabelSelector: "release=" + h.releaseName})
```

Comment:
suggestion: Just a thought but might be worth having a function or something that we call that just generates `"release="+h.releaseName` since it's used everywhere. Less chance of accidentally mispelling/creating wrong string in the future.